plugins/org.eclipse.osee.orcs.rest.model/src/org/eclipse/osee/orcs/rest/model/RelationEndpoint.java

```diff
@@ -91,5 +91,10 @@ Response createRelationByType(@PathParam("sideA") ArtifactId sideA, @PathParam("
    List<RelationTypeToken> convertRelations(@PathParam("artifactA") ArtifactId artifactA,
       @PathParam("oldRelationType") RelationTypeToken oldRelationType,
       @PathParam("newRelationType") RelationTypeToken newRelationType);
+   
+ @GET
```

Comment:
This is kinda weird spot to put this...

Reply:
With the removal of the DB Health Endpoint, this seemed the best spot.  It isn't currently used, so when we decide to start using it, we can move it if needed.